front/app/containers/EventsShowPage/components/InformationSectionMobile.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+// components
+import { Box } from '@citizenlab/cl2-component-library';
+
+// styling
+import { colors } from 'utils/styleUtils';
+
+// typing
+import { IEventData } from 'api/events/types';
+import EventDateStylized from './MetadataInformation/EventDateStylized';
+import Location from './MetadataInformation/Location';
+import FullEventTime from './MetadataInformation/EventTimeTextual';
+
+interface Props {
+  event: IEventData;
+}
+
+const InformationSectionMobile = ({ event }: Props) => {
```

Comment:
At the moment this is quite similar to the desktop view, but future designs will diverge quite a bit with regards to how information is shown on mobile vs desktop, so I wanted to create a separate component now.